assets/scss/base/_reused.scss

```diff
@@ -105,10 +66,6 @@
   gap: var(--spacing-5) var(--spacing-4);
 }
 
-.text-center {
-  text-align: center;
-}
-
 [data-reveal] {
   opacity: 0;
   transition: var(--transition-2);
```

Comment:
You're not passing which property to animate, so it falls back to `all`, which is not a good practice. You need to define whether it's eg. `color` or `opacity`